content/A good way to manage your Python projects using Pyenv and Pipenv.md

```diff
@@ -0,0 +1,290 @@
+Many combinations of compatibility issues can be found, but having those isolated environments you can avoid them. You will be able to install, uninstall, update, etc in one virtual environment without affecting the rest of them. As said, there are many tools to do this and in the next sections, I will talk about the ones that I've found pretty easy to understand and use.
+
+## Pyenv
+Pyenv is a tool used to manage different Python versions. With this tool you can:
+- Install several Python versions.
+- Set/change the global(default) Python version in your computer.
+- Set/change a Python version locally for a project.
+- Create and manage virtual environments.
+
+### Installation
+On macOS, you can install this tool using [Homebrew](https://brew.sh/):
+
+```
+brew install pyenv
+```
+
+To install it on other platforms, you can watch [this section in the GitHub docs](https://github.com/pyenv/pyenv#installation). After the installation, edit your used shell startup script (`.bashrc`, `.zshrc`, `.bash_profile`, ...) adding the following line:
+
+```
+if command -v pyenv 1>/dev/null 2>&1; then
+  eval "$(pyenv init -)"
+fi
+```
+
+Then reload your shell startup script. An example if you have `.bashrc`:
+
+```
+cd
+source .bashrc
+```
+
+That's it! You have installed Pyenv successfully.
+
+### Managing Python versions
+Now let's take a look at the possible Python versions that you can install through Pyenv, executing:
+
+```
+pyenv install --list
+```
+
+You will see a large list of Python versions. You can pick one to install it, let's say 3.9.1:
+
+```
+pyenv install 3.9.1
+```
+
+You can run that command with any version that you want to have installed on your computer. To see all the installed versions, just run:
+
+```
+pyenv versions
+```
+
+Also, you can set the global version that you want to use in your system, this will be the default one. For example, if you have already installed 3.8.8, you can set it as the global version running:
+
+```
+pyenv global 3.8.8
+```
+
+You can change that, of course, by executing the command again with the desired installed version. If you want to check your global version, just run:
+
+```
+python -V
+```
+
+With Pyenv, you can have a global and a local version for each project. To set a local version, go to the project folder in the terminal:
+
+```
+cd ~/path/to/the/project/folder
+```
+
+And set the local version, for example 3.9.1:
+
+```
+pyenv local 3.9.1
+```
+
+This will create a `.python-version` file in the folder indicating the current local Python version for the project. Also if you run `python -V` in that folder, you will see the local version, not the global one.
```

Comment:
```suggestion
This will create a `.python-version` file in the folder indicating the current local Python version for the project. Also, if you run `python -V` in that folder, you will see the local version, and not the global one.
```